packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -280,6 +281,15 @@ class RawAutocomplete<T extends Object> extends StatefulWidget {
   /// This parameter is ignored if [textEditingController] is defined.
   final TextEditingValue? initialValue;
 
+  /// {@template flutter.widgets.RawAutocomplete.optionsLayerLink}
+  /// Links the [optionsBuilder] to a custom [CompositedTransformTarget],
+  ///  enabling control over the positioning and size of the options overlay.
+  /// A [LayerLink] is required to change the location or size of the [optionsBuilder].
+  /// When setting [optionsLayerLink], [RawAutocomplete] must have a [CompositedTransformTarget]
+  /// as a parent widget to link the options overlay to the correct position in the widget tree.
```

Comment:
```suggestion
  /// Links the [optionsBuilder] to a custom [CompositedTransformTarget],
  /// enabling control over the positioning and size of the options overlay.
  ///
  /// A [LayerLink] is required to change the location or size of the [optionsBuilder].
  /// When setting [optionsLayerLink], [RawAutocomplete] must have a [CompositedTransformTarget]
  /// as a parent widget to link the options overlay to the correct position in the widget tree.
```

Also specify the behavior when this property is not provided.